pyproject.toml

```diff
@@ -120,7 +118,6 @@ match = '(?!(test_|_)).*\.py'
 
 [tool.pytest.ini_options]
 minversion = "6.0"
-addopts = "--verbose"
```

Comment:
Just to reduce the 1500 line terminal flood when running the starts to a grid of dots